graphql/availability.graphql

```diff
@@ -0,0 +1,38 @@
+extend type Query {
+    availabilities: [Availability!]! @paginate
```

Comment:
Don't think we are ever going to need this method - we will only query availability on a user(s)

Reply:
We'd still need a query for a user looking at their own availability?

Reply:
Nah - so in the future (thought we had already implemented but we haven't ) we will have a `currentUser` query that returns the authenticated user. Through the returned user type, you will be able to query the `availability` relationship like so:
```
query {
   currentUser: {
      availability: {
           to
           from
           ...
       }
   }
}
```

Reply:
Okay, removeed